Makefile

```diff
@@ -89,10 +91,12 @@ lib-test: ## Run library tests
 
 .PHONY: demo-build
 demo-build: ## Build the project (client, server and universal)
+	cd packages/server-dom-esbuild && npm run build;
```

Comment:
I prefer to keep commands working inside dune, so watch mode recompiles the thing. Can you move this inside demo/dune?